resources/js/controllers/fields/quill_controller.js

```diff
@@ -76,6 +76,25 @@ export default class extends Controller {
         ];
     }
 
+    containerCustomToolbar() {
+        const controlsGroup = {
+            "text":   ['bold', 'italic', 'underline', 'strike', 'link', 'clean'],
+            "color":  [{color: this.colors()}, {background: this.colors()}],
+            "header": [{header: '1'}, {header: '2'}],
+            "list":   [{list: 'ordered'}, {list: 'bullet'}],
+            "format": [{indent: '-1'}, {indent: '+1'}, {align: []}],
+            "media":  ['image', 'video'],
+        }
+        let container = [];
+
+        JSON.parse(this.data.get("toolbar")).forEach(function (element) {
+            if (element in controlsGroup)
+                container.push(controlsGroup[element])
+        });
+
+        return container
```

Comment:
Hi, I think we can write it down easier:

```javascript
JSON.parse(this.data.get("toolbar")).filter(element => element in controlsGroup);
```
We will not define a new variable and create a second array.
What do you think about this?

Reply:
Yes, this is better

Reply:
Ok, I ended up with this:
```js
return JSON.parse(this.data.get("toolbar"))
            .map(tool => controlsGroup[tool]);
```